packages/circuit-ui/components/SidePanel/SidePanel.module.css

```diff
@@ -4,15 +4,28 @@
   bottom: 0;
   left: unset;
   z-index: var(--cui-z-index-absolute);
+  background-color: var(--cui-bg-normal);
```

Comment:
Let's scope the normal background color to wide viewports where the side panel is on the same visual layer. On narrow viewports, it should keep using bg/elevated, since the side panel is rendered above the main content.